src/main/webapp/app/core/about-us/about-us-routing.module.ts

```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { AboutUsComponent } from 'app/core/about-us/about-us.component';
+
+const routes: Routes = [
+    {
+        path: 'about',
+        component: AboutUsComponent,
+        data: {
+            authorities: [],
+            pageTitle: 'aboutUs',
+        },
+    },
+];
+
+@NgModule({
+    imports: [RouterModule.forChild(routes)],
+    exports: [RouterModule],
+})
+export class AboutUsRoutingModule {}
```

Comment:
Please lazy-load this module OR make this part of another module we already have.

Reply:
Done. With lazy load

Reply:
Responsive layout also